test/harness/network.go

```diff
@@ -105,6 +106,24 @@ func (n *acceptanceTestNetwork) SendTransfer(nodeIndex int, amount uint64) chan
 	return ch
 }
 
+func (n *acceptanceTestNetwork) SendInvalidTransfer(nodeIndex int) chan *client.SendTransactionResponse {
```

Comment:
SendTransfer and SendInvalidTransfer are now mostly duplicate.. are we planning to add more of these? or does this mean we need to extract something outside to the caller?

Reply:
We will later refactor if needed

Reply:
Wait for 3 :)